formula-android-compose/src/main/java/com/instacart/formula/android/compose/ComposeViewFactory.kt

```diff
@@ -0,0 +1,32 @@
+package com.instacart.formula.android.compose
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.rxjava3.subscribeAsState
+import androidx.compose.ui.platform.ComposeView
+import com.instacart.formula.android.FeatureView
+import com.instacart.formula.android.ViewFactory
+
+
+abstract class ComposeViewFactory<RenderModel> : ViewFactory<RenderModel> {
+
+    override fun create(inflater: LayoutInflater, container: ViewGroup?): FeatureView<RenderModel> {
+        val view = ComposeView(inflater.context)
+        return FeatureView(
+            view = view,
+            bind = {
+                view.setContent {
+                    val model = it.observable.subscribeAsState(null).value
+                    if (model != null) {
```

Comment:
I have been wondering if that's going to be an issue (like a binking first frame).
Hopefully not, having to provide a "default" UI would be inconvernient
Reworking Formula so the RenderModelGenerator emits synchronously the first time would probable be a large effort

Reply:
Only legacy `RxFormula` might not have a default emission and we don't have too many of those. The `Formula` instances always emit and render model should already be available when `bind` is called. However, I'll confirm this behavior once I start testing the API.

Reply:
the proof will be in the pudding then 🤞

Reply:
I tested this with fragments and it works fine to skip the null value.